### PR TITLE
eth-bytecode-db: docker

### PR DESCRIPTION
**Pull request by rimrakhimov**
Add docker and docker compose file definitions

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 target
```
Comment (leviska) [inline .dockerignore:1]: Do we need this file? Does it do anything?

Comment (rimrakhimov) [reply .dockerignore:1]: Hmm, I cannot see where it is used. Probably, we can just remove it away

File: eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    container_name: 'eth-bytecode-db-verifier'
+    restart: always
+#    pull_policy: always
+    environment:
+      SMART_CONTRACT_VERIFIER__SERVER__HTTP__ENABLED: false
+      SMART_CONTRACT_VERIFIER__SERVER__GRPC__ENABLED: true
+      SMART_CONTRACT_VERIFIER__SERVER__GRPC__ADDR: 0.0.0.0:8044
+  eth-bytecode-db:
+    ## you can build an image locally, or use pre-built images from registry
+#    image: ghcr.io/blockscout/eth-bytecode-db:main
```
Comment (leviska) [inline eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml:25]: you can leave image _and_ build fields
https://docs.docker.com/compose/compose-file/build/#illustrative-sample

File: eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  database:
+    image: postgres:15
+    container_name: 'eth-bytecode-db-postgres'
```
Comment (leviska) [inline eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml:5]: usually container name is unspecified

File: eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - "8051:8051"
+    environment:
+      ## optional: if provided, would be used as a configuration file
+      ETH_BYTECODE_DB__CONFIG: /app/config.toml
+      ETH_BYTECODE_DB__VERIFIER__URI: http://smart-contract-verifier:8044
+      ETH_BYTECODE_DB__DATABASE__URL: postgres://postgres:@database:5432/eth_bytecode_db
+      ETH_BYTECODE_DB__DATABASE__RUN_MIGRATIONS: true
+    env_file:
+      ## optional: if provided, would overwrite values from configuration file
+      - ./config/base.env
```
Comment (leviska) [inline eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml:45]: if it's optional, then may be leave this commented?

Comment (leviska) [reply eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml:45]: so if somebody copypaste this, it will just work

**Review by leviska — APPROVED** [review]

**Pull request merged**